### PR TITLE
Remove unused attrs in `analysis/tests`

### PR DESCRIPTION
**Pull request by kkysen**
This removes unused attrs in `analysis/tests`.

### CODE REVIEW

**Comment by kkysen** [comment]
Note that these deletions have to be rebased into #749 when either lands in `master`.

**Review by aneksteind — APPROVED** [review]

**Comment by kkysen** [comment]
Also, this is a superset of #732, so I'd ideally like to get approval on that first before merging this.

**Pull request merged**
